### ISSUE TITLE
ResolVI.train() has its train_size parameter hardcoded

### ISSUE THREAD

**Issue by jberkh** [scverse/scvi-tools#3589]
**Is your feature request related to a problem? Please describe.**
In the `RESOLVI.train` method, `super().train` is called with a hardcoded `train_size=1.0`. This prevents the use of a validation set and effectively disables trainer parameters that rely on validation loss (e.g., early stopping).

**Describe the solution you'd like**
Unless there’s a specific reason for fixing train_size at 1.0, this parameter should be configurable.

A simple solution would be to either:
- Add `train_size` as an argument to the method and pass that, or
- Use `kwargs.get("train_size", 1.0)` when calling `super().train`.

**Comment by canergen** [comment]
Hi, the diffusion proportions per neighboring cell can't be amortized (are learned as a free parameter for each cell). This is required as the list of neighbors of each cell are not ordered. Thus using a validation set doesn't make sense. The prediction for these cells would be inaccurate. We don't recommend early stopping due to this limitation.
